index.js

```diff
@@ -9,6 +9,14 @@ const stew = require('broccoli-stew');
 module.exports = {
   name: 'ember-cli-typescript',
 
+  init() {
+    this._super.init.apply(this, arguments);
+
+    if (this.project.addons.includes(this)) {
```

Comment:
comments please

Reply:
Oops! The relevant comment lines are sitting safe and sound and un-staged in my local clone 🤦‍♂️